test/distributed/test_c10d_nccl.py

```diff
@@ -3977,10 +3977,10 @@ def test_timeout_dumps_on_idle_ranks(self):
                 self.assertEqual(t[0]['state'], 'completed')
             return
 
-        # Set heartbeat timeout to a shorter one (default timeout is 2 min).
-        os.environ[
-            "TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC"
-        ] = f"{NCCLTraceTestDumpOnTimeoutBase.timeout_sec * 2}"
+        # # Set heartbeat timeout to a shorter one (default timeout is 10 mins).
```

Comment:
intended to keep this commented out? or want to add it back?

Reply:
Hmm this should be deleted in the latest version.